### DISCUSSION TITLE
Clonezilla boot error

### DISCUSSION THREAD

**Discussion by lgwapnitsky** [ipxe/ipxe#567] [Q&A] [answered]
I've tried multiple downloads of Clonezilla, and have recompiled ipxe multiple times, yet keep winding up with the following error when booting:

![image](https://user-images.githubusercontent.com/647444/149847317-66e428eb-4e23-4705-8920-03c9f440d9d0.png)

This is my clonezilla config:

```
:clonezilla
kernel ${base-url}/clonezilla/live/vmlinuz
kernel ${base-url}/clonezilla/live/initrd.img
imgargs vmlinuz boot=live config noswap nolocales edd=on nomodeset ocs_live_run="ocs-live-general" ocs_live_extra_param="" ocs_live_keymap="" ocs_live_batch="no" ocs_daemonon="ssh" username=user ocs_lang="" vga=788 nosplash noprompt fetch=http://${base-url}/clonezilla/live/filesystem.squashfs
boot || goto failed
goto start

```

**Comment by NiKiZe** [comment, marked as answer]
Could you type out the error message? Including the URL? And also reading that page.

Is this pcbios or efi? And is it latest version of iPXE or something else?

See #564

**Reply by NiKiZe** [reply]
Oh, you have kernel United, which means it gets selected as boot image. Try using initrd instead.

**Reply by mcb30** [reply]
What @NiKiZe said.  Use the line:
```
initrd ${base-url}/clonezilla/live/initrd.img
```
i.e. use the `initrd` command instead of the `kernel` command to download the initrd.

**Comment by lgwapnitsky** [comment]
fat finger typo! thanks, guys!

*Larry G. Wapnitsky*


*E: ***@***.****
*Web: Larry.Wapnitsky.com <http://larry.wapnitsky.com/>Twitter:
@LGWapnitsky*
